instructor/common.py

```diff
@@ -1471,6 +1471,9 @@ def facs_analyze(request):
     """
     pk = request.session['assignment_id']
     assignment = models.Assignment.objects.get(id=pk)
+    facs, created = models.FlowCytometry.objects.get_or_create(
```

Comment:
well, you should write something like this then:
```
facs, _ = models.FlowCytometry.objects.get_or_create
```
unless you use `_` for something else